recipes/jkqtplotter/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "jkqtplotter" %}
+{% set version = "2019.11.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jkriege2/JKQtPlotter/archive/v{{ version }}.tar.gz
+  sha256: 94d39834c6f4e4afd4ad8511231a26d74fd88dc60d7b3ccd85b7ce00f32ff487
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - qt
+    - cmake
+    - make
```

Comment:
Use this only for `unix`

Reply:
Fixed in f42e85fcf53a37bc58481d7fe05b7228bc750aa8.